### ISSUE TITLE
Unable to reproduce the results on fold classification task

### ISSUE THREAD

**Issue by huangtinglin** [a-r-j/ProteinWorkshop#91]
Hi! Thanks for this great work. I'm attempting to replicate the results of Table 2 presented in the paper, but I've noticed that the performance is different from what was reported. I applied the EGNN on the fold_classification tasks without using any auxiliary tasks. The command I used is shown below:

```
python proteinworkshop/train.py encoder=egnn task=multiclass_graph_classification dataset=fold_fold features=ca_seq +aux_task=none
```

The performance of each test set in my experiment is fold: 0.28, family: 0.91, super_family: 0.378:

<img width="1722" alt="image" src="https://github.com/a-r-j/ProteinWorkshop/assets/26502648/6fffdd4c-8e31-4077-bf58-f3e1b627531b">

Please let me know if there is any information I need to provide. Thanks!

**Comment by a-r-j** [comment]
Hi, looks like you're missing some hparams:

`optimiser.optimizer.lr=${hparams.hparams.lr} decoder.residue_type.dropout=${hparams.hparams.decoder_dropout}`

You can also look at our sweep configs to make sure LR schedulers, and early stopping etc are configured appropriately.

**Comment by huangtinglin** [comment]
Thanks! Let me try tuning some hyperparameters.
